packages/eslint-plugin/package.json

```
@@ -17,7 +17,7 @@
     "url": "https://github.com/BuilderIO/mitosis"
   },
   "scripts": {
-    "test": "jest",
+    "test": "jest --watch",
```

`test` is used by our CI. If it has `--watch` I think this means our CI tests will never pass.

```suggestion
    "test": "jest",
    "test:watch": "jest --watch",
```